src/transformers/models/t5/modeling_t5.py

```diff
@@ -640,6 +640,11 @@ def forward(
         hidden_states, present_key_value_state = self_attention_outputs[:2]
         attention_outputs = self_attention_outputs[2:]  # Keep self-attention outputs and relative position weights
 
+        # clamp inf values
```

Comment:
maybe improve comment slightly:

```suggestion
        # clamp inf values to enable fp16 training
```